### PR TITLE
Clarification of intent and code of conduct.

### PR DESCRIPTION
**Pull request by tef**
I've added in the status quo, expanded on some issues (we don't ticket or name badge), and edited the code of conduct to look less patch work and design by committee. I've also rephrased some troublesome sections.

Going to let people look at this a little for typos or problems before merging, but i've been getting positive feedback on IRC. I'm open to wording and choice, but I'm pretty stubborn about what computer is meant to be and how I've been running it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,39 +14,28 @@
 
       <h4>The Plan</h4>
 
-      <p>This might be the group for you if you want to meet socially conscious nerds to talk about interesting things. This is not an entrepreneurial meetup, nor is it networking: It is a support group. </p>
-      <p>Although education and outreach are both important to us, the primary goal is to create a social group for people in and around tech, from all backgrounds, where they feel comfortable and welcome.</p>
-
       <ul>
         <li>We go to a good pub or venue.</li>
-        <li>We eat food.</li>
+        <li>We eat food, We drink, be it beer, cocktails or soft drinks.</li>
```
Comment (ntlk) [inline index.html]: second 'we' doesn't need uppercase

Comment (didlix) [inline index.html]: :+1:

**Comment by timcowlishaw** [comment]
An emphatic :thumbsup: from me.

**Comment by seubert** [comment]
+1

**Comment by ntlk** [comment]
I think this is good, and the feedback on IRC was also good, so I am going to merge it in.

File: index.html [hunk 2]
```diff
@@ -14,39 +14,28 @@
 
       <h4>The Plan</h4>
 
-      <p>This might be the group for you if you want to meet socially conscious nerds to talk about interesting things. This is not an entrepreneurial meetup, nor is it networking: It is a support group. </p>
-      <p>Although education and outreach are both important to us, the primary goal is to create a social group for people in and around tech, from all backgrounds, where they feel comfortable and welcome.</p>
-
       <ul>
         <li>We go to a good pub or venue.</li>
-        <li>We eat food.</li>
+        <li>We eat food, we drink, be it beer, cocktails or soft drinks.</li>
```
Comment (DRMacIver) [inline index.html:19]: Perhaps "We eat and we drink (be it beer, cocktails or soft drinks)" ?

**Comment by didlix** [comment]
Why are these commandments anyway? Are they adding value?

**Pull request merged**